### PR TITLE
Update example config file with new modules

### PR DESCRIPTION
**Pull request by ptomato**
There are new modules in the Endless SDK, so we should update the
example config file to allow hacking on them.

https://phabricator.endlessm.com/T21272

### CODE REVIEW

File: example.flapjack.ini [hunk 1]
```diff
@@ -76,12 +76,16 @@ add_extensions = org.gnome.Sdk.Docs//3.26:share/runtime/docs
 # This is the list of modules that you want to be able to open and hack on.
 
 modules =
-    xapian-glib
     eos-metrics
```
Comment (smspillaz) [inline example.flapjack.ini:79]: Seems unrelated?

**Review by smspillaz — COMMENTED** [review]
Other than that looks good

File: example.flapjack.ini (hunk 1 above)
Comment (ptomato) [reply example.flapjack.ini:79]: I just took all the editable modules in the order they were in the manifest, so I think this change is good, but I'll update the commit message to explain.

Comment (smspillaz) [reply example.flapjack.ini:79]: Okay makes sense

**Review by smspillaz — APPROVED** [review]
